site/src/pages/components/focusgroup.explainer.mdx

```diff
@@ -419,10 +424,10 @@ focusgroups).
 
 | HTML attribute value        | CSS property &amp; value                           | Explanation                                                                                                                                   |
 | --------------------------- | -------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
-| focusgroup="horizontal"     | focus&#8209;group&#8209;direction:&nbsp;horizontal | The focusgroup items will respond to forward and backward movement only with the "horizontal" arrow keys (left and right).                    |
-| focusgroup="vertical"       | focus&#8209;group&#8209;direction:&nbsp;vertical   | The focusgroup items will respond to forward and backward movement only with the "vertical" arrow keys (up and down).                         |
-| focusgroup="block"          | focus&#8209;group&#8209;direction:&nbsp;block      | The focusgroup items will respond to forward and backward movement only with the "vertical" arrow keys or "horizontal" based on writing mode. |
-| focusgroup="inline"         | focus&#8209;group&#8209;direction:&nbsp;inline     | The focusgroup items will respond to forward and backward movement only with the "horizontal" arrow keys or "vertical" based on writing mode. |
+| focusgroup="inline"         | focus&#8209;group&#8209;direction:&nbsp;inline     | The focusgroup items will respond to forward and backward movement only with arrow keys that are parallel to this element's "inline" axis (e.g., left and right arrow keys for `horizontal-tb` writing mode). |
+| focusgroup="block"          | focus&#8209;group&#8209;direction:&nbsp;block      | The focusgroup items will respond to forward and backward movement only with arrow keys that are parallel to this element's "block" axis (e.g., up and down arrow keys for `vertical-*` writing modes). |
+| focusgroup="horizontal"     | focus&#8209;group&#8209;direction:&nbsp;horizontal | The focusgroup items will respond only to left and right arrow keys (regardless of content directionality). ⚠️This value is **NOT RECOMMENDED** for best i18n experiences. Consider using `inline` instead. |
```

Comment:
since this is a brand new API; should we not introduce physical properties and just stick to logical?

Reply:
I was on the fence about that, but since you brought it up, yes, let's just support logical properties!

Reply:
Dropped